src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Color[] COLORS = Color.values();
+    private final Random random;
```

Comment:
The `random` object is used only within the `getRandomColor` method and does not need to be a class-level field. You can declare it as a local variable inside the method.